docs/setup/docker.md

```diff
@@ -65,16 +59,18 @@ volumes:
   postgres-db:
 ```
 
-Then, run the following command to provision the images:
+Notice that the values for `EWALLET_SECRET_KEY` and `LOCAL_LEDGER_SECRET_KEY` are missing. You can generate these secret keys by running:
 
-```bash
-$ docker-compose up
 ```
+$ docker run --rm omisego/ewallet:latest elixir -e "IO.puts 32 |> :crypto.strong_rand_bytes() |> Base.encode64()"
```

Comment:
Might worth generating keys with `openssl rand -base64 32` instead so user don't need to have Elixir install, or have a running instance of eWallet (which is the recommended way to do so in [Helm chart](https://github.com/omisego/charts/blob/3033de464aaea4210af4d23fb1972c03aab239ed/ewallet/values.yaml#L58)).

Reply:
That's a very good point, realised that it was kinda of annoying for people to have to download Elixir just for this.